modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java

```diff
@@ -690,9 +690,9 @@ private String getNameUsingEnumPropertyNaming(String name) {
             case original:
                 return name;
             case camelCase:
-                return camelize(name, true);
+                return camelize(StringUtils.lowerCase(name), true);
```

Comment:
I think this breaks the behavior if e.g. `name` is `someEnumValue`, which would incorrectly result in `someenumvalue`, as opposed to `someEnumValue`